rotkehlchen/tests/unit/globaldb/test_asset_updates.py

```diff
@@ -530,23 +542,23 @@ def test_conflict_updates(assets_updater: AssetsUpdater, globaldb: GlobalDBHandl
 *"""  # noqa: E501
     update_patch = mock_asset_updates(
         original_requests_get=requests.get,
-        latest=2,
-        updates={'1': {
+        latest=16,
```

Comment:
same as before why the rename?

Reply:
the test was using an arbitrary update version but now that we're enforcing versions they need to be changed.